app/src/main/java/ar/com/wolox/android/example/ui/home/HomeModule.kt

```diff
@@ -0,0 +1,14 @@
+package ar.com.wolox.android.example.ui.home
+
+import dagger.Module
+import dagger.android.ContributesAndroidInjector
+
+@Module
+abstract class HomeModule {
+
+    @ContributesAndroidInjector
+    protected abstract fun homeActivity(): HomeActivity?
```

Comment:
You can remove the '?' after the homeActivity (and every single kotlin module LoL)